docs/user-interface/handlers/customize.md

```diff
@@ -1,122 +1,316 @@
 
-Handlers expose the native control that implements the .NET MAUI cross-platform control via the `NativeView` property. This property can be accessed to set platform properties, invoke platform methods, and subscribe to platform events.
+Each handler class exposes the native control that implements the cross-platform control via its `NativeView` property. This property can be accessed to set native control properties, invoke native control methods, and subscribe to native control events. In addition, the cross-platform control implemented by the handler is exposed via its `VirtualView` property.
 
-Handlers are global and therefore customization can occur anywhere in your .NET MAUI app. For example, customizing a handler for a control in your `App` class will result in all controls of that type being customized in the app. Similarly, customizing a handler for a control in the first page of your app will also result in all controls of that type in the app being customized.
+Handlers can be customized to augment the appearance and behavior of a .NET MAUI control beyond the customization that's possible through the control's API. Handlers are global and therefore customization can occur anywhere in your .NET MAUI app. For example, customizing a handler for a control in your `App` class will result in all controls of that type being customized in the app. Similarly, customizing a handler for a control in the first page of your app will also result in all controls of that type in the app being customized.
 
-## Examples
+Handlers can be customized per platform by using compiler preprocessor directives, to multi-target code based on the platform. Alternatively, you can use partial classes to organize your code into platform-specific folders and files. For more information about conditional compilation, see [Conditional compilation](/dotnet/csharp/language-reference/preprocessor-directives#conditional-compilation).
 
-Handlers are typically customized to augment the appearance and behavior of a native platform control. Handlers can be customized per platform by using the `#ifdef` compiler directive, to multi-target code based on the platform. However, you can just as easily use platform-specific folders or files to organize such code.
+## Handler lifecycle
 
-> [!NOTE]
-> Handler customization can range from the simple to the complex. More complex examples will appear in time.
+All handler-based .NET MAUI controls support two handler lifecycle events:
 
-### Customize the background color of all controls
+- `HandlerChanged` is fired after the handler for a cross-platform control has been created. This event indicates that the native control that implements the cross-platform control is available, and all the property values set on the cross-platform control have been applied to the native control.
```

Comment:
You've used `fired` and `fires` a lot 😀 but the proper term for events is `raised`. The code raises the event or the event was raised. Code doesn't fire events and the event wasn't fired.